frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpanded.tsx

```diff
@@ -171,10 +166,6 @@ const ImageSection: React.FC<{
         alt={resource?.image.alt ?? ""}
       />
     )
-  } else if (resource) {
-    return (
-      <Image src={DEFAULT_RESOURCE_IMG} aspect={config.width / config.height} />
-    )
```

Comment:
We need this to display the default image for resources that have a `null` image (or we get the loading skeleton), unless I'm missing something?